### ISSUE TITLE
Changing @imported files does not trigger a rebuild

### ISSUE THREAD

**Issue by Kurre** [tailwindcss/tailwindcss#130]
It seems that some changes in v0.1.4 broke the `watch` feature (at least for me).

So basically, Laravel Mix, PostCSS (postcss-import, tailwindcss), non-Laravel project 

With `v0.1.4` and `yarn watch`:
- Change `component.css` -> CSS is not compiled
- Change `postcss.css` -> CSS is compiled

With `v0.1.3` and `yarn watch`:
- Change `component.css` -> CSS is compiled
- Change `postcss.css` -> CSS is compiled

I made a small [demo project](https://github.com/Kurre/tailwindcss-watch-issue) with minimal amount of setup.
Hopefully you are able to reproduce this issue.

Other specs/info:
- Windows 10
- `node -v`: `v8.9.0`
- `yarn -v`: `1.3.2`
- `tailwind.js` unmodified

PS. Haven't tried with `npm`, but it probably doesn't make any difference (at least, it shouldn't make)

**Comment by adamwathan** [comment]
Updated the title to explain the situation a bit more clearly.

You're right, I can reproduce this on my machine and I'm not sure what the cause is. It's unfortunate because in 0.1.3, changing `tailwind.js` would not trigger rebuilds, but in `0.1.4` it does. Something has caused imported files to stop triggering rebuilds though and I have no idea why; I can't see how the changes we made could have caused it.

Will keep looking into this deeper.

For reference, here's the change I'm assuming affected this:

https://github.com/tailwindcss/tailwindcss/pull/77/files#diff-1fdf421c05c1140f6d71444ea2b27638
